docs/question/html.md

```diff
@@ -120,18 +139,35 @@ adding the `mathjax_ignore` class to an HTML element.
 
 ## Rendering panels from `question.html`
 
-When a question is displayed to a student, there are three "panels" that will be shown at different stages: the "question" panel, the "submission" panel, and the "answer" panel. These display the question prompt, the solution provided by the student, and the correct answer.
+When a question is displayed to a student, there are three "panels" that will be shown at different stages: the `"question"` panel, the `"submission"` panel, and the `"answer"` panel. These display the question prompt, the solution provided by the student, and the correct answer.
+
+The `"question"` panel is displayed when the question is first shown to the student. The `"submission"` panel is displayed after the student submits an answer (but before they finish attempting the question), and the `"answer"` panel is displayed when the student either submits the correct answer or runs out of attempts. You can see the [lifecycle diagram](index.md#lifecycle) for more details on how these panels are displayed.
 
 All three panels display the same `question.html` template, but elements will render differently in each panel. For example, the `<pl-number-input>` element displays an input box in the "question" panel, the submitted answer in the "submissions" panel, and the correct answer in the "answer" panel.
 
 Text in `question.html` can be set to only display in the "question" panel by wrapping it in the `<pl-question-panel>` element. This is useful for the question prompt, which doesn't need to be repeated in the "submission" and "answer" panels. There are also elements that only render in the other two panels.
 
 ## Hiding staff comments in `question.html`
 
-Please note that HTML or JavaScript comments in your `question.html` source may be visible to students in the rendered page source. To leave small maintenance notes to staff in your `question.html` source, you may prefer to use a Mustache comment that will stay hidden. Please refer to [this FAQ item](../faq.md#how-can-i-add-comments-in-my-questionhtml-source-that-wont-be-visible-to-students).
+HTML or JavaScript comments in your `question.html` source are visible to students in the rendered page source. To leave small maintenance notes to staff in your `question.html` source, you should use a Mustache comment `{{! ... }}` that won't be rendered in the final HTML.
+
+To prevent students from seeing staff comments, you can use [Mustache comments](https://mustache.github.io/mustache.5.html#Comments) that will be removed during the rendering process. Never put sensitive information, such as solutions, in a HTML/JS comment.
+
+Example:
+
+```html
+<!-- This is an HTML comment. It will not be visible to students in the web page, 
+ but *will be included* in the rendered page source, so students may be able to
+ see it by reading the HTML source. -->
+{{! This is a Mustache comment. It will NOT be shown in the rendered page source. }}
+```
 
 ## How questions are rendered
```

Comment:
I think this entire section could be removed -- 99% of instructors shouldn't care about this.

Reply:
Absolutely, I put this in here back when we introduced the new renderer. A few people probably looked at this then, but it's not relevant anymore. I may just remove this in another PR to keep this diff cleaner.

Reply:
I opened https://github.com/PrairieLearn/PrairieLearn/pull/11835 to remove it independently. Having it in a separate diff should make it slightly easier to find it in the future if we ever need it again.